### ISSUE TITLE
GHDOG will reach 1B market capitalization

### ISSUE THREAD

**Issue by Tarryco** [thisrepo/dog#47]
Small innovations can bring in massive amounts of traffic.

**Comment by httpscure** [comment]
you know if this can hits millions in the middle of this nuts meta im sure it will catch more eyes. i think need to create telegram and some activities to push this up

**Comment by Tarryco** [comment]
Yes, we need Telegram Group. at the same time there should be an official website, which domain do you guys think is better?

![image](https://github.com/user-attachments/assets/e2458763-0804-41f4-bd29-c6f65bb470d7)

**Comment by httpscure** [comment]
fine with anything bro xD

**Comment by ghdog-sol** [comment]
![image](https://github.com/user-attachments/assets/3d91c507-6d28-42da-8f81-20ed966854eb)

**Comment by binance-hk** [comment]
ghdog.org, would you all agree?

**Comment by Cryptobertl** [comment]
I created one already I will just finish to setup and post the link please be a bit patient I am in a lovely hotel 5 stars having breakfast with my wife that’s prior anytime
